src/yb/server/default-path-handlers.cc

```diff
@@ -74,7 +74,8 @@
 #include "yb/util/version_info.h"
 #include "yb/util/version_info.pb.h"
 
-DEFINE_int64(web_log_bytes, 1024 * 1024,
+DEFINE_int64(
+    web_log_bytes, 1024 * 1024,
```

Comment:
why do we have this change?

Reply:
Apologies for the unrelated changes, it was the auto-formatter that made those changes, I will revert the changes back 😅